pkg/cloud/cloud.go

```diff
@@ -1407,6 +1414,22 @@ func isAWSErrorBlockDeviceInUse(err error) bool {
 	return false
 }
 
+// isAWSErrorModifyVolumeInvalidParameter returns a boolean indicating whether the
+// given error is caused by invalid parameters in a ModifyVolume request
+func isAWSErrorModifyVolumeInvalidParameter(err error) bool {
+	if isAWSError(err, "InvalidParameterValue") {
```

Comment:
Should we check for `InvalidParameter`, `InvalidParameterDependency`, `UnsupportedOperation`, and `UnknownParameter` as well? 

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html#api-error-codes-table-server

Reply:
The three included error codes are the only ones that EC2 ModifyVolume returns (Seems that EC2 APIs aren't standardized in that way, and that error code table does not reflect which APIs return which error)

But if we're generalizing the function to `isAWSErrorInvalidParameter` then yes!